### ISSUE TITLE
Modifying Features in mobile (Android) is not smooth 

### ISSUE THREAD

**Issue by Shubham98Tiwari** [openlayers/openlayers#10991]
**Describe the bug**
I am working on an ionic application and implementing geometry drawing using Openlayers 6 the problem is that when I try to modify the the geometry, it is working well in browser (Desktop) but it hangs in mobile. Here is my code:

```
loadMap()
  {
this.source = new VectorSource();
var vector = new VectorLayer({
  source: this.source,
  style: new Style({
    fill: new Fill({
      color: 'rgba(255, 255, 255, 0.2)'
    }),
    stroke: new Stroke({
      color: '#ffcc33',
      width: 2
    }),
    image: new CircleStyle({
      radius: 7,
      fill: new Fill({
        color: '#ffcc33'
      })
    })
  })
});

  let view = new OlView({
      center: fromLonLat([74.643235, 25.347955]),
      zoom: 18// 5
      });

  var select = new Select({
    wrapX: false
  });

  var modify = new Modify({
    features: select.getFeatures(),
    style: new Style({
      image: 

      new RegularShape({
          fill: new Fill({
              color: 'red'
          }),
          points: 4,
          radius1: 15,
          radius2: 1
      })
  })
  });

  setTimeout(() => {
    this.map = new OlMap({
    interactions: defaultInteractions().extend([select, modify]),
    target: 'map',
    view: view,
    layers: [
      new OlTileLayer({
        preload: 3,
        source: new OlOSM(),
      }),
      vector
    ],
    loadTilesWhileAnimating: true,
    });  
  },0); 
}

  drawingMode(event){
    this.geometry = event.detail.value;
    this.map.removeInteraction(this.draw);
    this.map.removeInteraction(this.snap);
    this.addInteractions();
    }

    addInteractions() {
      if(this.geometry !== "None"){
      this.draw = new Draw({
        source: this.source,
        type: this.geometry,
        style: new Style({
          image: 

          new RegularShape({
              fill: new Fill({
                  color: 'red'
              }),
              points: 4,
              radius1: 15,
              radius2: 1
          }),
          stroke: new Stroke({
              color: 'blue',
              width: 5
          }),
          fill: new Fill({
              color: "rgba(187, 211, 247, 0.6)"
          })
      })
      });
        this.map.addInteraction(this.draw);
        this.snap = new Snap({source: this.source});
        this.map.addInteraction(this.snap);

        var self = this;
        this.draw.on('drawend', function(event) {
          self.map.removeInteraction(self.draw);
          self.map.removeInteraction(self.snap);
        });     
       }
```
**To Reproduce**
1. Go to [Draw and Modify Features](https://openlayers.org/en/latest/examples/draw-and-modify-features.html) in your **mobile phone browser** or in desktop go to this link  then inspect it (by pressing f12 or ctrl+shift+I) and toggle device toolbar on.
2. Then draw a LineString, polygon or any other feature.
3. Then try to modify the feature drawn.
4. While modifying you will feel the hanging and freezing, like if you will pull the vertex it will not work as smoother as it works in browser (desktop while toggle device toolbar is off ).    

**Expected behavior**
The editing should be as smoother as  it works in browser (desktop while toggle device toolbar is off ).

**Comment by neylsongularte** [comment]
+1

**Comment by neylsongularte** [comment]
Modifying Features in mobile is not working on 6.2.0 version and above

**Comment by neylsongularte** [comment]
apparently resolved on 6.3.2-dev.1589843798021
